ci/docker/install/centos7_install_ccache.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Script to build ccache for centos7 based images
+
+set -ex
+
+pushd .
+
+yum -y install epel-release
```

Comment:
Please make sure you're only installing necessary dependencies. It seems like quite a lot are duplicate or just simply don't fit here. Some examples would be git, wget, make, asciidoc, gcc. For the others, I'm not sure what they exactly do.

Reply:
Everything listed here is needed for an isolated ccache build from source.

Reply:
See discussion below.